### ISSUE TITLE
The guardian function get_user_perm always returns an empty list on the ASL Signbank

### ISSUE THREAD

**Issue by Woseseltops** [Signbank/Global-signbank#465]
As a result of this, the check for the correct dataset permissions always fails, and everybody gets the message 'You are not authorized to change the selected dataset.' when they try to create a new sign. 

For now, the dataset permission check has been commented out (and not committed), but obviously we need a better solution). The most important question is: why is this list always empty, whatever I do in the admin and dataset management tools?

**Comment by Woseseltops** [comment]
Okay, fixed it. The problem/confusion was:
* This guardian function only checks for permission at the object level (in this case: the permissions set for the single ASL dataset). When I created this issue, there weren't any.
* On top of that, according to the documentation, the function returns 'only those [permissions] assigned directly for the user.'

I now added that it will also check get_group_perms, and now it seems to work as intended.
